src/builtin.js

```diff
@@ -360,41 +228,39 @@ Sk.builtin.min = function min($default, key, args) {
             }
         }
     } else {
-        let lowest_compare = Sk.misceval.callsimOrSuspendArray(key, [lowest]);
+        let lowest_compare = Sk.misceval.callsimArray(key, [lowest]);
```

Comment:
No. I wrote the original pr without knowing what suspensions were!

My understanding now is that misceval.callsimOrSuspendArray just allows a a suspension to be returned. But it doesn’t handle the suspension. 

Misceval.chain is used to actually handle a retuned suspension. And since we don’t use chain here we can’t handle a suspension. 

I can rewrite this algorithm as separate pr to handle suspensions? now that I know what they are...!

Reply:
I have a pr for this see #1119 
I will merge those changes into this branch when/if they are merged into master.